EditorsDraft/index.html

```diff
@@ -1375,17 +1424,23 @@
      "repeatFrequency": "P1W",
      "byDay": "http://schema.org/Wednesday",
      "startTime": "19:00Z",
-     "endTime": "20:00Z"
+     "endTime": "20:00Z",
```

Comment:
This is incorrect and should be resolved: see https://github.com/openactive/data-model-validator/issues/99 - the timezone component doesn't make any sense in this context. A Yoga session would be Tuesday 7pm in the winter, and Tuesday 8pm in the summer.